src/modules/tas_studio/editor/mod.rs

```diff
@@ -934,22 +1006,93 @@ impl Editor {
+                            // Mode to know which to switch to.
+                            // For left or right, it would be TargetAndEnd so we can turn better.
+                            // For left-right and yaw, it would bet Alt where we can change the
+                            // first yaw field value.
+                            let (mode, adjustment_dir, alt) =
+                                if let Some(AutoMovement::Strafe(StrafeSettings { dir, .. })) =
+                                    bulk.auto_actions.movement
+                                {
+                                    // Match StrafiDir so we can get adjustment mode and
+                                    // adjustment_dir.
+                                    // Then match StrafeType to get values for alt.
+                                    match dir {
+                                    StrafeDir::Left | StrafeDir::Right | StrafeDir::Best => {
+                                        let adjustment_dir = if matches!(dir, StrafeDir::Right) {
+                                            -adjustment_dir
+                                        } else {
+                                            adjustment_dir
+                                        };
+
+                                        (
+                                            MaxAccelYawOffsetMode::StartAndTarget,
+                                            adjustment_dir,
+                                            MaxAccelYawoffsetAlt::None,
+                                        )
+                                    }
+                                    StrafeDir::Yaw(yaw) | StrafeDir::Line { yaw } => (
+                                        MaxAccelYawOffsetMode::Alt,
+                                        adjustment_dir,
+                                        MaxAccelYawoffsetAlt::Yaw(yaw),
+                                    ),
+                                    StrafeDir::LeftRight(count) | StrafeDir::RightLeft(count) => {
+                                        let adjustment_dir =
+                                            if matches!(dir, StrafeDir::RightLeft(_)) {
+                                                -adjustment_dir
+                                            } else {
+                                                adjustment_dir
+                                            };
+
+                                        (
+                                            MaxAccelYawOffsetMode::Alt,
+                                            adjustment_dir,
+                                            MaxAccelYawoffsetAlt::LeftRight(count),
+                                        )
+                                    }
+                                    _ => return Err(ManualOpError::UserError(
+                                        "Max accel yaw offset does not support current strafe dir."
```

Comment:
Obsolete?

Reply:
Point strafing isn't supported for editor operations because that would adds some more complexity (and effort). I think the message is misinformed. I will change that.